__tests__/app.test.js

```diff
@@ -51,4 +51,50 @@ describe("3.5. GET /api", () => {
                 expect(body).toEqual(json);
             });
     });
+});
+
+
+
+
+
+
+
+
+describe("5. GET /api/reviews", () => {
+    it("200: Should respond with an array of review objects", () => {
+        return request(app)
+            .get("/api/reviews")
+            .expect(200)
+            .then(({ body: reviews }) => {
+                const reviewsArr = reviews.reviews;
+                expect(reviewsArr).toHaveLength(13);
+                reviewsArr.forEach((review) => {
+                    expect(review).toEqual(
+                        expect.objectContaining({
+                            owner: expect.any(String),
+                            title: expect.any(String),
+                            review_id: expect.any(Number),
+                            category: expect.any(String),
+                            review_img_url: expect.any(String),
+                            created_at: expect.any(String),
+                            votes: expect.any(Number),
+                            designer: expect.any(String),
+                            comment_count: expect.any(Number),
```

Comment:
We should also add a test to check the comment_count specifically for one object and ensure the value is as expected